readme.md

```diff
@@ -94,12 +94,12 @@ ipc.answerMain('get-emoji', async emojiName => {
 
 ```js
 const electron = require('electron');
-const ipc = require('electron-better-ipc');
+const {ipcMain: ipc} = require('electron-better-ipc');
 
-const win = electron.BrowserWindow.getFocusedWindow();
+const window = electron.BrowserWindow.getFocusedWindow();
```

Comment:
We can't use `window` as it shadows the `window` global. However, we could use `browserWindow`.

Reply:
Why is that a problem?

Reply:
Users might confuse it with the `window` global when in the renderer context (which is just a browser context) if `window` is used far from its declaration.